### DISCUSSION TITLE
Set default views in the config file

### DISCUSSION THREAD

**Discussion by dievilz** [orhun/gpg-tui#43] [Ideas]
It would be nice if I could set in the config file, let's say, a detail level of 3 (full details) for all keys (not just the one selected) for both public and secret keys, as default every time I use the program.

**Comment by orhun** [comment]
This is implemented in 849d90df3a51a33c0fb2b863988282d8f645c5b3

Now you can set the detail level via command-line arguments:

```sh
gpg-tui --detail-level full
```

or via configuration file:

```toml
[general]
  detail_level = "full"
```

**Reply by dievilz** [reply]
Great!!!! Keep it up man!
